### DISCUSSION TITLE
412 Error: authentication failed

### DISCUSSION THREAD

**Discussion by Huremovic95** [emailjs-com/help#4] [Q&A]
When i want to make a email service i get this error -> 412iCloud: Invalid login: 535 5.7.8 Error: authentication failed.
My email and password are correct, whats wrong?

**Comment by Huremovic95** [comment]
fixed it by not checking the Send test email to verify configuration box. Dont know how and why but if someone has the same problem this worked for me.

**Comment by xr0master** [comment]
Unchecking "Send test email to verify configuration" allows you to save the service AS IS. In some cases, it can be useful, mainly for transactional services.

However, it is very strange that this could somehow solve the issue with iCloud. Most likely, you forgot to create the 3rd part app password. We strongly discourage using a master password. Moreover, it won't work if you have multiple authentication enabled for the provider.

**Comment by dfents** [comment]
I was seemingly able to fix the issue send test service by changing Region from Australia to Worldwide
